src/v0.1.0/properties/structures/_cheminfo_smarts_query.yaml

```diff
@@ -0,0 +1,21 @@
+$$schema: "https://schemas.optimade.org/meta/v1.2/optimade/property_definition"
+$id: "https://schemas.optimade.org/namespaces/cheminformatics/v0.1/properties/structures/_cheminfo_smarts_query"
+title: "A chemical substructure query in the SMARTS language"
+x-optimade-type: "string"
+x-optimade-definition:
+  kind: "property"
+  version: "0.1.0"
+  format: "1.2"
+  name: "_cheminfo_smarts_query"
+  label: "_cheminfo_smarts_query"
+type:
+  - "string"
+  - "null"
+description: |-
+  A property used to specify chemical substructure queries in the SMILES Arbitrary Target Specification (SMARTS) language, defined in the Daylight SMARTS documentation (https://www.daylight.com/dayhtml/doc/theory/theory.smarts.html).
+  This property is not intended to be stored or returned by the API, but rather to be used in queries to search for structures containing a specified substructure via the `CONTAINS` operator.
+examples:
+  - "C(=O)O.OCC>>C(=O)OCC.O"
```

Comment:
Strictly speaking, this is a "reactions SMARTS" which might not make too much sense in the substructure search (I might be wrong). Maybe we can remove this example?

Reply:
I think after the discussion yesterday we're now basically removing this field and adding the description here to the SMILES field...